### DISCUSSION TITLE
No private key password?

### DISCUSSION THREAD

**Discussion by racer8** [Chia-Network/chia-blockchain#5716] [General]
Is there a reason the private key is not password protected like other wallets?  I do not like relying on Windows security.

**Comment by CryptoBlockchainTechnologies** [comment]
> Is there a reason the private key is not password protected like other wallets? I do not like relying on Windows security.

I know the devs are busy working on pooling but this seems like a very simple implementation. While there are ways to create a new wallet, copy the mnemonic to a safe place, and then delete the wallet. Most people are not doing this and are relying on the trust of their system security. Being only one click away from the mnemonic password without a password is not ideal

**Comment by pluto2021** [comment]
Try using Mac. You would need to enter admin password to access the mnemonic keys. So it might be a good idea to use a Mac with a separate wallet to keep your XCH safe for the moment until a hardware cold wallet is available.
